src/datreant/core/rsync.py

```diff
@@ -0,0 +1,46 @@
+"""Rsync wrapper"""
+
+import subprocess
+import os
+
+
+def rsync(source, dest, compress=True, backup=False, dry=False,
+          include=None, exclude=None, rsync_path='/usr/bin/rsync'):
+    opts = ['-r']  # Recursive
+
+    if compress:
+        opts.append('-c')
```

Comment:
From the `rsync` man page: isn't this the option for a checksum? Do we intend to use `-z` here?